Source/DBAlertController.swift

```diff
@@ -38,12 +38,12 @@ public class DBAlertController: UIAlertController {
 // In the case of view controller-based status bar style, make sure we use the same style for our view controller
 private class DBClearViewController: UIViewController {
     
-    private override func preferredStatusBarStyle() -> UIStatusBarStyle {
-        return UIApplication.sharedApplication().statusBarStyle
+    fileprivate override var preferredStatusBarStyle : UIStatusBarStyle {
```

Comment:
Can you remove the space before the colon? I try to follow the Ray Wenderlich style guide. (https://github.com/raywenderlich/swift-style-guide#spacing)